src/login/login.cpp

```diff
@@ -66,7 +66,15 @@ struct epoll_event login_lobbyviewEpollEvent = { EPOLLIN };
 #include "login.h"
 #include "login_auth.h"
 #include "login_conf.h"
+#include "message_server.h"
 
+/* wolfSSL */
```

Comment:
All of this old stuff can die. Do the current login/lobby files need to exist anymore?

Reply:
File deleted